cypress/integration/like.spec.ts

```diff
@@ -0,0 +1,24 @@
+it('likes an episode then finds it in the library', () => {
+  cy.visitAndWaitForHydration('/podcasts/1');
+
+  // Find a card and give it a name
+  cy.get('[data-cy="episode-card"]').first().as('episodeCard');
```

Comment:
praise: But certainly mad props for adding an integration test to begin with. I was thinking we'll have to go back to it at the end of the project

Reply:
I understand this and I agree, but I couldn't find a way to semantically identify the elements uniquely without also depending on the HTML structure. Like, if I `findByRole('p')`, which one should it pick, the title or duration? And there may not always be ARIA attributes that make sense. I also definitely didn't want to depend on the CSS classes. If you can suggest a better way to find the element semantically without using `data-cy`, I'd gladly implement it.

Reply:
Here's a helpful [_timestamped_ video](https://youtu.be/F8LH9d9eN3M?t=2486) of a person testing a similar application in Cypress with the Testing Library.

TL;DW:
* There's no way to identify a container by role, so our best bet is to use the `findByTestId` (and we can configure the Testing Library to search for the `data-cy` attribute to avoid writing out the CSS selector like that).
* Once you get ahold of the container, you can use semantic queries, mimic the behaviour of a user — first, ensure that the text of the sought episode title is present in the card, then find a button (`role="button"`) that has a "like" in the name and click that

So basically, I propose the following changes: 
1. Configure Cypress Testing Library to use `data-cy` attributes for the `findByTestId` query (but if I'm being honest, I'd prefer `data-test-id`, it's much clearer to me)
2. Change the way we locate the button to a `findByRole` query
3. Avoid `expect(title).to.be.a('string');`, because that doesn't concern the user, instead use the query `findByText` and assert its existence

Reply:
@VanishMax let me know what you think of this